src/lib/operators/get_table.cpp

```diff
@@ -239,8 +239,38 @@ std::shared_ptr<const Table> GetTable::_on_execute() {
     ++output_chunks_iter;
   }
 
+  // Check if a table index is irrelevant because of pruning.
+  const auto check_if_index_is_irrelevant_lambda = [this](const std::shared_ptr<AbstractTableIndex>& table_index) {
+    // Check if indexed ColumnID has been pruned.
+    const auto indexed_column_id = table_index->get_indexed_column_id();
+    if (std::find(_pruned_column_ids.begin(), _pruned_column_ids.end(), indexed_column_id) !=
+      _pruned_column_ids.end()) {
+      return true;
+    }
+
+    const auto indexed_chunk_ids = table_index->get_indexed_chunk_ids();
+
+    // Early out if index is empty.
+    if (indexed_chunk_ids.empty()) {
+      return false;
+    }
+
+    auto indexed_chunk_ids_vector = std::vector<ChunkID>(indexed_chunk_ids.begin(), indexed_chunk_ids.end());
```

Comment:
We usually don't put the name of the used language construct (here: vector) into the variable name. `indexed_chunk_ids` is sufficient.